### PR TITLE
fix: design review changes

### PR DESCRIPTION
**Pull request by aronshamash**
- as per https://nidigitalsolutions.jira.com/browse/REPLAT-5586

### CODE REVIEW

File: packages/fixture-generator/src/mock-slice.ts [hunk 1]
```diff
@@ -169,9 +161,13 @@ function mockSecondaryOneAndColumnistSlice(): SecondaryOneAndColumnistSliceWithN
 
 function mockSecondaryOneAndFourSlice(): SecondaryOneAndFourSliceWithName {
   const tiles = getTiles(5);
+  const secondaryTile = {
+    ...tiles[0],
+    article: { ...tiles[0].article, strapline: "Readers share their top tips" }
```
Comment (tuncaulubilge) [inline packages/fixture-generator/src/mock-slice.ts]: Article doesn't have a strapline field, the tile does

File: packages/edition-slices/src/tiles/tile-n/index.js [hunk 2]
```diff
@@ -17,9 +17,8 @@ const TileN = ({ onPress, tile }) => (
         flagColour={styles.flagColour}
         headlineStyle={styles.headline}
         labelColour={colours.functional.greyLabel}
+        strapline={tile.article.strapline}
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/tile-n/index.js]: This should be tile.strapline as strapline only exists within the tile

**Comment by nehasri89** [comment]
LG apart from tunca's comments 👍

**Comment by aronshamash** [comment]
@nehasri89 updated `strapline` now. Please have another look.

**Pull request merged**